# **Nicolás Nahuel Moreyra**

  <img src="imgs/photo_myself.jpeg" style="width: 309px; height: 431px; position: center" >

## **Personal information**

+ Date of birth: October 4th, 1989
+ Nationality: Argentinean
+ E-Mail Personal: [niconm89@gmail.com](mailto:niconm89@gmail.com)
+ E-Mail Laboral: [nmoreyra@ege.fcen.uba.ar](mailto:nmoreyra@ege.fcen.uba.ar)
  
### **Social/Scientific Networks**

+ Linkedin: [Nicolás Moreyra](https://www.linkedin.com/in/nicolasmoreyra/)
+ GitHub: [Nicolás Moreyra](https://github.com/niconm89)
+ Google Academics: https://scholar.google.com/citations?hl=es&user=2ygu-Z8AAAAJ
+ ORCID: https://orcid.org/0000-0003-0451-2399

### **About me**

I have a BSc. in Bioinformatics and a Ph.D. in Biological Sciences oriented to evolutionary biology. I am passionate about next generation sequencing technologies, omic sciences and programming to answer biological questions.

## **Education**
  | Name                                                              | Institution                |    Project title             | Start Date | End Date |
  |-------------------------------------------------------------------|----------------------------|------------------------------|------------|----------|
| BSc. in Bioinformatics | Facultdad de Ingeniería ([FI](http://ingenieria.uner.edu.ar/)), Universidad Nacional de Entre Ríos ([UNER](https://www.uner.edu.ar/)). Oro Verde, Entre Rios, Argentina.  | Deconvolution of ELISPOT T-cell responses against HIV using immunoinformatic screening methods. |   2008   |   2014   |
| Ph.D. in Biological Sciences | Facultad de Ciencias Exactas y Naturales ([FCEyN](https://exactas.uba.ar/)), Universidad de Buenos Aires ([UBA](https://www.uba.ar/)). Ciudad de Buenos Aires, Argentina. | Genome assembly and comparative analysis of the species D. borborema, D. antonietae y D. koepferae (Drosophila buzzatii cluster, repleta group).  |   2015   |   2020   |
| Postdoctoral Studies        | Instituto de Ecología, Genética y Evolución de Buenos Aires ([IEGEBA](https://www.ege.fcen.uba.ar/institucional/iegeba/), UBA-CONICET) and Fundación para el Estudio de Especies Envasivas ([FuEDEI](https://fuedei.org/)). Ciudad de Buenos Aires, Argentina.    | Comparative genomics of cactophilic moths of the genus Cactoblastis.  | 2021          | Present      |

  
 ### **Research Scholarships**
  | Description + Funding  | Project title                    | Place of Work                              | Start Date| End Date |
  |----------------------------|----------------------------------|--------------------------------------------|-----------|----------|
| Ph.D. scholarship (Type I) - Agencia Nacional de Promoción Científica y Tecnológica ([ANPCyT](https://www.argentina.gob.ar/ciencia/agencia)) | Genome assembly of four South American cactophilic species belonging the Drosophila buzzatii cluster | [IEGEBA](https://www.ege.fcen.uba.ar/institucional/iegeba) | 2015          | 2018       |
| Short-term scholarship of DEANN Project “Marie Curie Actions - International Research Staff Exchange Scheme (IRSES)” financed by the European Commision. | Identification and description of transponsable elements families in stressful cactophilic Drosophila genomes. | [Gonzalez Lab](https://www.biologiaevolutiva.org/gonzalez_lab/), Institut de Biologia Evolutiva (IBE), Barcelona, Cataluña, Spain.          | 2016          | 2016       |
| Short-term Internship - Consejo Nacional de Investigaciones Científicas y Técnicas ([CONICET](https://www.conicet.gov.ar/)) and [ANPCyT](https://www.argentina.gob.ar/ciencia/agencia) | Collaboration under the project “Genome assembly of four South American cactophilic species belonging the Drosophila buzzatii cluster”. |  [Matzkin Lab](https://matzkinlab.org/), University of Arizona (UA), Tucson, United States (US)  | 2018          | 2018     |
| Ph.D. Scholarship (Type II) - [CONICET](https://www.conicet.gov.ar/) | Genome assembly of four South American cactophilic species belonging the Drosophila buzzatii cluster | [IEGEBA](https://www.ege.fcen.uba.ar/institucional/iegeba/) | 2018          | 2021     |
| Postdoctoral Scholarship - [CONICET](https://www.conicet.gov.ar/) | Genome assembly of four South American cactophilic species belonging the Drosophila buzzatii cluster | [IEGEBA](https://www.ege.fcen.uba.ar/institucional/iegeba/) and [FuEDEI](https://fuedei.org/) | 2021          | 2024     |

 ### **Publications, preprints and drafts**
 + Rondón JJ*, Pisarenco VA, Rozas J, **Moreyra NN**, Hurtado J, Hasson E (2022). Evolution of the Odorant-Binding Protein gene family in Drosophila. In preparation to be submitted to [Frontiers](https://www.frontiersin.org/research-topics/27610/diptera-chemical-ecology-in-the-genomic-era-from-genes-to-behavior?j=1847727&sfmc_sub=337759798&l=94_HTML&u=48422372&mid=7236711&jb=51).
 + Mensch J*, **Moreyra NN***, Hasson E (2022). Transcriptional responses of immature and mature females acclimated under the same low temperature provides insights into cold tolerance of cactophilic Drosophila buzzatii. In preparation to be submitted to [Genes](https://www.mdpi.com/journal/genes/special_issues/Arthropods_Evolution).
 + **Moreyra NN***, Almeida F, Allan C, Frankel N, Matzkin L, Hasson E (2022). Phylogenomics provides insights into the evolution of cactophily and host plant shifts adaptation in Drosophila. In preparation to be submitted to [Elife](https://elifesciences.org/).
 + **Moreyra NN***, Mensch J, Hurtado J, Almeida F, Laprida C, Hasson E (2019). What does mitogenomics tell us about the evolutionary history of the Drosophila buzzatii cluster (repleta group)? PLoS ONE 14(11): e0220676. https://doi.org/10.1371/journal.pone.0220676.
 + Shome S*, Parra RG, Fatima N, Monzon AM, Cuypers B, Moosa Y, Coimbra NDR, Assis J, Giner-Delgado C, Dönertaş HM, Cuesta-Astroz Y, Saarunya G, Allali I, Gupta S, Srivastava A, Kalsan M, Valdivia C, J Olguin-Orellana G, Papadimitriou S, Parisi D, Kristensen NP, Rib L, Guebila MB, Bauer E, Zaffaroni G, Bekkar A, Ashano E, Paladin L, Necci M, **Moreyra NN**, Rydén M, Villalobos-Solís J, Papadopoulos N, Rafael C, Karakulak T, Kaya Y, Gladbach Y, Dhanda SK, Šoštarić N, Alex A, DeBlasio D, Rahman F. (2019). Global network of computational biology communities: ISCB's Regional Student Groups breaking barriers. F1000Research, 8, ISCB Comm J-1574. https://doi.org/10.12688/f1000research.20408.1.
 + Fiorella Cravero*, Lionel Uran Landaburu, **Nicolás N. Moreyra**, Emilio Fenoy, Carla L Padilla Franzotti, Esteban Lanzarotti, R. Gonzalo Parra*, Nicolas Palopoli*, Alexander M. Monzon* (2018) 2nd Argentine Symposium of Young Bioinformatics Researchers (2SAJIB) organized by the ISCB-SC RSG-Argentina. PeerJ Preprints 6:e3504v1.
 + Parra RG*, Defelipe LA, Guzovsky AB, Monzon AM, Cravero F, Mancini E, **Moreyra N**, Padilla Franzotti CL, Revuelta MV, Freiberger MI, Gonzalez NN, Gonzalez GA, Orts F, Stocchi N, Hasenahuer MA, Teppa E, Zea DJ, Palopoli N. (2016) Highlights of the 1st Argentine Symposium of Young Bioinformatics Researchers (1SAJIB) organized by the ISCB RSG-Argentina. PeerJ Preprints 4:e2494v1.

 ### **Awards**
 + 2019. Travel Fellowship of the European Molecular Biology Organization (EMBO)
  Description: fellowship to assist to the EMBO Practical Course “Bioinformatics and Genome Analyses”.
  Place: Fondazione Edmund Mach (FEM), San Michelle all'Adige, Trento, Italy.
 + 2016. Travel Fellowship Award of the European Conference on Computational Biology to assist to the ECCB2016. 
  Place: The Hague, Netherlands. 
 + 2013. Fellowship Award “Jóvenes Profesionales TICs 2013”. 
  Project title: Development of a bioinformatic tool for the analysis of eating behavior in different social groups. ANPCyT, Argentina.

 ### **Technical Skills**

#### Experince working in omic-related fields:
  + UNIX/Linux systems user
  + Experience working in HCP servers.
  + High-throuput sequencing data QC and curation (single-end, pair-ends, mate-pairs and long reads)
  + Repeat library generation and Transposable Element annotation
  + Genome assembly and annotation
  + Gene annotation curation (e.g. Apollo)
  + Orthology search/prediction
  + Management of genomic files such as gff, fasta, bed and related formats
  + Phylogenomics: phylogeny reconstruction and divergence time estimation
  + Molecular evolution analysis
  + Transcriptome assembly
  + Differential expresion analysis
  + Sequencing Experimental Design
  + Metagenomics (beginner)

#### Programming Languajes and Frameworks:
  + Python
  + Bash
  + AWK
  + R
  + Perl
  + C++ (bassic)
  
#### Other Knowledge:
  + Collaborative work on GitHub and GitLab
  

## **Work Experience**

+ Maths and English private teacher
+ National and regional fencing referee
+ Youth centre monitor
+ Radio advertisements marker at ArceMedia
+ Web designer y Web master for:
    + [Pepu Alcalde](http://pepualcalde.org/)
    + [Tecrima](https://www.tecrima.com/)
+ Waiter at Covanegra Restaurant(Capdepera, Mallorca)
+ Software Engineer Intern at Avanade S.L.U: Jan. 2020 - May. 2020
    + During the internship it was mostly training in web technologies such as AngularJS.
+ Junior Developer at Aghasa Turis: Jun. 2020 - Present
    + Currently my functions consist of the development and implementation of web and desktop applications using web technologies such as Node.js, ExpressJS, Vue.js, etc...

## **Other Info**

 - Eagerness to learn and face new challenges
 - Versatil and tenacious
 - Ability to work in teams as well as to carry out autonomous projects
 - Communication and social skills
 - Commitment, responsibility and determination to my duties
 - Pro-active
 - Sociable
 - Ability to create practical solutions
